projects/plugins/jetpack/modules/widgets/social-media-icons.php

```diff
@@ -1,23 +1,24 @@
-<?php
-/*
-Plugin Name: Social Media Icons Widget
-Description: A simple widget that displays social media icons
-Author: Automattic Inc.
-
-This widget is now deprecated.
-Any new features should go into modules/widgets/social-icons.php instead.
-@see https://github.com/Automattic/jetpack/pull/8498
-
-*/
-
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName.php
+/**
+ * Social Media Icons Widget
+ *
+ * This widget is now deprecated.
+ * Any new features should go into modules/widgets/social-icons.php instead.
+ *
+ * @see https://github.com/Automattic/jetpack/pull/8498
+ *
+ * @package automattic/jetpack
+ */
 
 /**
  * WPCOM_social_media_icons_widget class.
  *
  * @extends WP_Widget
+ *
+ * phpcs:disable PEAR.NamingConventions.ValidClassName.Invalid
  */
 class WPCOM_social_media_icons_widget extends WP_Widget {
-
+	// phpcs:enable PEAR.NamingConventions.ValidClassName.Invalid
```

Comment:
This had to be split instead of just a single `// phpcs:ignore` because otherwise it triggers another phpcs linting error.